### PR TITLE
Add tags functionality for youtube channels to the website.

### PR DESCRIPTION
**Pull request by hruntov**

### CODE REVIEW

File: youtube_base/youtubers/views.py [hunk 1]
```diff
@@ -148,24 +149,44 @@ def post(self, request, *args, **kwargs):
 
         """
         form = CategoryForm(request.POST)
+        tag_form = TagForm(request.POST)
+
         if form.is_valid():
             categories = form.cleaned_data['categories']
             youtubers = Youtuber.objects.filter(categories__in=categories)
-
```
Comment (Yevhenii-Yatchenko) [inline youtube_base/youtubers/views.py:154]: I'd recommend not to do changes + refactorings within a single commit.

1. Refactoring commit.
2. Additional changes

Or visa versa.

File: tests/api_test.py [hunk 2]
```diff
@@ -28,10 +28,10 @@ def test_extracting_data_from_url_type_2(self):
 
     def test_extracting_data_from_url_type_3(self):
         """The third type of URL is in the format: "https://www.youtube.com/c/@{username}"""
-        api = YoutubeApi("https://www.youtube.com/c/@EnglishClass101")
```
Comment (Yevhenii-Yatchenko) [inline tests/api_test.py:31]: BTW, why it was unstable?

**Review by Yevhenii-Yatchenko — APPROVED** [review]

File: tests/api_test.py (hunk 2 above)
Comment (hruntov) [reply tests/api_test.py:31]: I don't mind. Now in the GitHub Actions container, the YouTube API returns a different channel id for that channel. Maybe it somehow depends on the location of the server.

Comment (hruntov) [reply tests/api_test.py:31]: I am surprised that one YouTube channel can have several IDs.

**Pull request merged**